kroxylicious-operator/pom.xml

```diff
@@ -57,6 +57,11 @@
     </dependencyManagement>
 
     <dependencies>
+        <dependency>
```

Comment:
Should we be declaring the module in the bom?

Reply:
I've considered that a few times and generally think no, as the BOM is really for filter authors looking for a stable Java API to build against. I have also wondered about a separate Operator BOM, but its not clear to me who the intended consumer of that would be. 

So for the moment I don't think it should be in the bom.